### DISCUSSION TITLE
Why is UserType column part of the default output for Get-MgUser?

### DISCUSSION THREAD

**Discussion by alexandair** [microsoftgraph/msgraph-sdk-powershell#1754] [General]
Get-MgUser doesn't output the UserType property by default.
Why is then the UserType column part of the default output table?
That column is always empty by default.

**Comment by peombwa** [comment]
`UserType` is a property of `User` type. As per the [user resource type reference doc](https://learn.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0#properties), `UserType` is :
> A string value that can be used to classify user types in your directory, such as Member and Guest.

> Returned only on $select. Supports $filter (eq, ne, not, in, and eq on null values). NOTE: For more information about the permissions for member and guest users, see [What are the default user permissions in Azure Active Directory?](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/users-default-permissions?context=graph/context#member-and-guest-users)

``` powershell
➜ Get-MgUser -Select UserType, DisplayName | select UserType, DisplayName

UserType DisplayName
-------- -----------
Member   Conf Room Adams
Member   AdeleV
Member   MOD Administrator
Member   AlexW
Member   AllanD
Member   Automate Bot
```

**Reply by alexandair** [reply]
My question remains:
Why is UserType part of the DEFAULT output?
As you cannot get it unless you specify -Select, it shouldn't be present by default because it's always empty (by default).

**Reply by peombwa** [reply]
Aha! I see your point. It is currently included in `format.ps1xml` as part of the default view because it returned by default in beta. We didn't have a way to distinguish `format.ps1xml` by API versions in v1.x of the module. Everything was bundled together in one module.

We will review the existing default views in v2 since we now have separate modules for `v1.0` and `beta`. I've opened an issue to track this at https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/1903.
